src/freenet/clients/fcp/RequestStatus.java

```diff
@@ -63,10 +69,15 @@ synchronized void restart(boolean started) {
 		this.totalBlocks = total;
 		this.minBlocks = min;
 		this.fetchedBlocks = fetched;
+        // clone() because Date is mutable
+		this.latestSuccess
```

Comment:
I am aware of this, it is due to Eclipse bugs.

I had tried to manually fix the resulting inconsistencies during the work on plugin-fcp-rewrite, but this showed that it usually takes more time than writing the actual code. This is because Eclipse sometimes automatically converts tabs/spaces when pasting or pressing "tab", so editing the wrong whitespace out is not easy.

I can upgrade my Eclipse to see whether this is fixed, but I would be glad if I could postpone this until my bachelor thesis is over (3 months). I don't think it would be a good idea to modify my working environment while the thesis is running.

Merging such code meanwhile should not wreak much havoc: We used to recommend people to configure their IDE to 4-spaces-per-tab, and with that this won't be visible. It is only visible here because Github uses something else than 4-spaces-per-tab.

-> ACK to ignore such stuff for 3 months?